grails-app/services/au/org/ala/biocache/hubs/WebServicesService.groovy

```diff
@@ -659,6 +659,32 @@ class WebServicesService {
         dataQualityCodes
     }
 
+    // maps from country name to iso code
```

Comment:
Please add proper Javadoc - `/** Foo */` so IDE can display info about method from the controller code, etc...